pkgs/development/tools/sigrok-firmware-fx2lafw/default.nix

```
@@ -1,21 +1,23 @@
 { lib
 , stdenv
-, fetchurl
+, fetchgit
+, autoreconfHook
 , sdcc
 }:
 
 stdenv.mkDerivation rec {
   pname = "sigrok-firmware-fx2lafw";
-  version = "0.1.7";
+  version = "unstable-2023-08-15";
 
-  src = fetchurl {
-    url = "https://sigrok.org/download/source/sigrok-firmware-fx2lafw/sigrok-firmware-fx2lafw-${version}.tar.gz";
-    sha256 = "sha256-o/RA1qhSpG4sXRmfwcjk2s0Aa8BODVV2KY7lXQVqzjs=";
+  src = fetchgit {
+    url = "git://sigrok.org/${pname}";
```

```suggestion
    url = "git://sigrok.org/sigrok-firmware-fx2lafw";
```
a change to pname likely will make this wrong